### ISSUE TITLE
Why my loss is lower?

### ISSUE THREAD

**Issue by lt573152873** [ElvishElvis/68-Retinaface-Pytorch-version#7]
I use LS3D-W dataset , when I training, the loss is lower ..  expamle:  total_loss : 1.1131469011306763    classification: 0.70500928      bbox: 0.07543718814849854    landmark: 0.6654008626937866  .
Check whether the fault is rectified.
